UK Import fails on feature with empty name.

Starting from a clean install of mapit I’m unable to import the boundary line data because the import fails when encountering a feature with an empty name.

For the build I’m using a Makefile always starting from a clean database so that it’s reproducible. I’ve also tried to download a the boundary line file again from https://osdatahub.os.uk/downloads/open/BoundaryLine to check whether that got somehow corrupted.

Here is the sha256 hash for verification:

```
$ sha256sum /tmp/Downloads/bdline_essh_gb.zip 
6ddfc8ae05dce11e2b9401963a3bc9874369383f58a441b42052d1c1855b3625  /tmp/Downloads/bdline_essh_gb.zip
```

Here is the output from the failing build:

```
.venv/bin/python mapit/manage.py mapit_UK_import_boundary_line --control=mapit_gb.controls.first-gss --commit unpack/bdline/Data/GB/county_electoral_division_region.shp unpack/bdline/Data/GB/county_region.shp unpack/bdline/Data/GB/district_borough_unitary_region.shp unpack/bdline/Data/GB/district_borough_unitary_ward_region.shp unpack/bdline/Data/GB/european_region_region.shp unpack/bdline/Data/GB/greater_london_const_region.shp unpack/bdline/Data/GB/parish_region.shp unpack/bdline/Data/GB/scotland_and_wales_const_region.shp unpack/bdline/Data/GB/scotland_and_wales_region.shp unpack/bdline/Data/GB/unitary_electoral_division_region.shp unpack/bdline/Data/GB/westminster_const_region.shp
…
unpack/bdline/Data/GB/parish_region.shp
…
…
  New area: CPC E04013024 176675 Shipley CP
Traceback (most recent call last):
  File "mapit/manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/roman/mapit-build/.venv/lib/python3.8/site-packages/django/core/management/__init__.py", line 401, in execute_from_command_line
    utility.execute()
  File "/home/roman/mapit-build/.venv/lib/python3.8/site-packages/django/core/management/__init__.py", line 395, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/roman/mapit-build/.venv/lib/python3.8/site-packages/django/core/management/base.py", line 330, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/roman/mapit-build/.venv/lib/python3.8/site-packages/django/core/management/base.py", line 371, in execute
    output = self.handle(*args, **options)
  File "/home/roman/mapit-build/.venv/lib/python3.8/site-packages/django/core/management/base.py", line 544, in handle
    label_output = self.handle_label(label, **options)
  File "/home/roman/mapit-build/mapit/mapit_gb/management/commands/mapit_UK_import_boundary_line.py", line 54, in handle_label
    name = name.decode('iso-8859-1')
AttributeError: 'NoneType' object has no attribute 'decode'
make: *** [Makefile:100: targets/import-bdline] Error 1
```

--- a/mapit_gb/management/commands/mapit_UK_import_boundary_line.py
+++ b/mapit_gb/management/commands/mapit_UK_import_boundary_line.py
@@ -49,7 +49,7 @@ class Command(LabelCommand):
         ds = DataSource(filename)
         layer = ds[0]
         for feat in layer:
-            name = feat['NAME'].value
+            name = feat['NAME'].value or ''
             if not isinstance(name, six.text_type):
                 name = name.decode('iso-8859-1')
 

